explorer/rest/tests/test_rest.py

```diff
@@ -102,29 +108,29 @@ def _assert_get_api_nem_blocks(client, expected_code, expected_result, **query_p
 	response = _get_api_nem_blocks(client, **query_params)
 
 	# Assert:
-	assert expected_code == response.status_code
 	assert expected_result == response.json
+	_assert_status_code_and_headers(response, expected_code)
```

Comment:
try to be consistent with naming wrt `expected_code` and `expected_status_code` throughout file